### PR TITLE
[Mailer][Mime] add `DraftEmail` documentation

### LINKED ISSUE #16309: [Mime] add DraftEmail
| Q            | A
| ------------ | ---
| Feature PR   | symfony/symfony#44311
| PR author(s) | @kbond
| Merged in    | 6.1

### PR DESCRIPTION
**Pull request by kbond**
Closes #16309.

### CODE REVIEW

File: mailer.rst [hunk 1]
```diff
@@ -1266,6 +1266,46 @@ The following transports only support tags:
...
+the ``X-Unsent`` header. Many email clients can open these files and interpret
+them as *draft emails*. You can use these to create advanced ``mailto:`` links.
+
+Here's an example of making one available to download::
+
+    use Symfony\Component\HttpFoundation\Response;
+    use Symfony\Component\HttpFoundation\ResponseHeaderBag;
+    use Symfony\Component\Mime\DraftEmail;
+
+    $content = (new DraftEmail())
```
Comment (94noni) [inline mailer.rst]:
```suggestion
   $message = (new DraftEmail())
```

?
this var seems not used
also there are 2 `->toString()` maybe one is to much

File: mailer.rst [hunk 2]
```diff
@@ -1266,6 +1266,46 @@ The following transports only support tags:
...
+
+    $response = new Response($message->toString());
+    $contentDisposition = $response->headers->makeDisposition(
+        ResponseHeaderBag::DISPOSITION_ATTACHMENT,
+        'download.eml'
+    );
+    $response->headers->set('Content-Type', 'message/rfc822');
+    $response->headers->set('Content-Disposition', $contentDisposition);
+
+    return $response;
```
Comment (94noni) [inline mailer.rst]: I would at least encapsulate in a proper controller action example :)

File: mailer.rst [hunk 3]
```diff
@@ -1266,6 +1266,46 @@ The following transports only support tags:
 
 * OhMySMTP
 
+Draft Emails
+------------
+
+.. versionadded:: 6.1
+
+    ``Symfony\Component\Mime\DraftEmail`` was introduced in ``symfony/mime`` 6.1.
```
Comment (94noni) [inline mailer.rst]: perhaps remove the ``symfony/mime``, I've never seen such package desc inside versionadded code block before

**Review by 94noni — APPROVED** [review]

**Comment by javiereguiluz** [comment]
Nice new feature! Thanks Kevin for working on this and for also providing the docs!

**Pull request merged**